manager/src/main/java/com/redhat/service/smartevents/manager/api/user/BridgesAPI.java

```diff
@@ -90,6 +96,10 @@ public Response createBridge(@Valid BridgeRequest bridgeRequest) {
         String customerId = identityResolver.resolve(jwt);
         String organisationId = identityResolver.resolveOrganisationId(jwt);
         String owner = identityResolver.resolveOwner(jwt);
+
```

Comment:
When service limit is reached (i.e. max number of Bridges), the error is returned. However the error is not formatted properly as an error type - only text is returned.

Expected error format:
```
{
  "kind": "Error",
  "id": "4",
  "href": "/api/v1/errors/4",
  "code": "OPENBRIDGE-4",
  "reason": "wqweqwer"
}
```

Reply:
You will need to map `ServiceLimitExceedException` in [`exceptioninfo.json`](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/main/infra/src/main/resources/exception/exceptionInfo.json).

Reply:
thanks @manstis

Reply:
Done.
Error response 
```
{
    "kind": "Error",
    "id": "31",
    "href": "/api/v1/errors/31",
    "code": "OPENBRIDGE-31",
    "reason": "Max allowed bridge instance limit exceed"
}
```